lightning/src/ln/channel.rs

```diff
@@ -848,7 +851,7 @@ impl<ChanSigner: ChannelKeys> Channel<ChanSigner> {
 		let mut txouts: Vec<(TxOut, Option<(HTLCOutputInCommitment, Option<&HTLCSource>)>)> = Vec::with_capacity(self.pending_inbound_htlcs.len() + self.pending_outbound_htlcs.len() + 2);
 		let mut included_dust_htlcs: Vec<(HTLCOutputInCommitment, Option<&HTLCSource>)> = Vec::new();
 
-		let dust_limit_satoshis = if local { self.our_dust_limit_satoshis } else { self.their_dust_limit_satoshis };
+		let holder_dust_limit_satoshis = if local { self.holder_dust_limit_satoshis } else { self.counterparty_dust_limit_satoshis };
```

Comment:
This is confusing - holder_dust_limit_satoshis may be counterparty_dust_limit_satoshis.

Reply:
I believe this (and below) should be broadcaster.

Reply:
Fixed, and reference below too. FYI I've a commit simplifying code on my anchor branch : 0a9430d